library/classes/thumbnail/Thumbnail.class.php

```diff
@@ -32,6 +32,12 @@ class Thumbnail
      */
     public function __construct($max_size = null)
     {
+        //validate that GD extension was enabled
+        if (!extension_loaded('gd'))
+        {
+            die('Abort '.basename(__FILE__).' : Missing DG extensions');
```

Comment:
Yes. he see it - 
![image](https://cloud.githubusercontent.com/assets/17809866/20310571/262ab8f4-ab55-11e6-95ad-1e7c7ead65a7.png)